test/config_afl.ml

```diff
@@ -0,0 +1,11 @@
+let () =
+  let fd = Unix.openfile Sys.argv.(1) Unix.[O_RDONLY] 0 in
+  let {Unix.st_size = len ; _ } = Unix.fstat fd in
+  let input =
+    let b = Bytes.create len in (* "best effort": *)
+    Bytes.sub_string b 0 (Unix.read fd b 0 len) in
```

Comment:
putting the `in` on the next line with indentation = 2 spaces makes it much easier to read imho...